### PR TITLE
Fix some issues

### LINKED ISSUE #99: admin2: input boxes should disable input
**Describe the bug**
Input boxes still trigger binds

**To Reproduce**
1. `bind 1 say hello`
2. Click on Server tab
3. Edit one of the fields and type in `1`
4. See `hello` being sent in the chat

**Expected behavior**
`guiSetInputMode('no_binds_when_editing')` or similar

### LINKED ISSUE #100: admin2: server properties should prevent invalid input
**Describe the bug**
It is possible to type "hello" in the "game speed" value (e.g.)

**To Reproduce**
1. Go to Server tab
2. Click on Game Speed edit
3. Type "hello"

**Expected behavior**
Only numbers should be allowed

**Screenshots**
image

### LINKED ISSUE #101: admin2: input validation for server props
**Describe the bug**
Typing nothing or letters into server props cause a server error.

**To Reproduce**
1. Go to server tab
2. Click on game speed
3. Type `hello` or empty the box
4. Click `set`

**Expected behavior**
`Set` should either be disabled OR input validation is performed and the operation is blocked when you click `Set`.

**Additional context**
Related to #100 so inputting `hello` should not be possible. Only empty inputs need to be checked. Alternatively an empty input can set it to the default.

### PR DESCRIPTION
**Pull request by FileEX**
***Summary***
- Fixes #99, fixes #100, fixes #101, improves #104 
- During edit fields characters other than numbers has been disabled
- Enabled glitches, properties checkboxes and 'Set' buton for heat haze
- Added more properties to refresh function and added also glitches to refresh function
- Added validate if value of field is numer, empty field sets default value
- Added missing default value for field (eg. weather, fps)
- Unused code has been commented

### CODE REVIEW

File: [admin]/admin2/server/admin_server.lua [hunk 1]
```diff
@@ -263,6 +263,15 @@ addEventHandler(
     end
 )
 
+addEvent("aServerRefresh", true)
+addEventHandler(
+    "aServerRefresh",
+    root,
+    function()
+        triggerClientEvent("aClientRefresh", client, isGlitchEnabled("quickreload"), isGlitchEnabled("fastmove"), isGlitchEnabled("fastfire"), isGlitchEnabled("crouchbug"), isGlitchEnabled("highcloserangedamage"), isGlitchEnabled("hitanim"), isGlitchEnabled("fastsprint"), isGlitchEnabled("baddrivebyhitbox"), isGlitchEnabled("quickstand"))
```
Comment (qaisjp) [inline [admin]/admin2/server/admin_server.lua]: Might be useful for us to just add `isGlitchEnabled` clientside

File: [admin]/admin2/client/main/admin.lua [hunk 2]
```diff
@@ -42,6 +42,8 @@ function aAdminMain.Open()
         guiCreateLabel(0.75, 0.05, 0.45, 0.04, "Admin Panel by lil_Toady", true, aAdminMain.Form)
         aAdminMain.Panel = guiCreateTabPanel(0.01, 0.05, 0.98, 0.95, true, aAdminMain.Form)
 
+        guiSetInputMode('no_binds_when_editing')
```
Comment (qaisjp) [inline [admin]/admin2/client/main/admin.lua]: This line fixes #148

File: [admin]/admin2/server/admin_server.lua (hunk 1 above)
Comment (FileEX) [reply [admin]/admin2/server/admin_server.lua]: Sure, but now `isGlitchEnabled` is only serverside function.

**Comment by CrosRoad95** [comment]
i don't think one pr with few fixes should be accepted, what if one of you fix goes wrong and will cause issues elsewhere?

**Comment by FileEX** [comment]
> i don't think one pr with few fixes should be accepted, what if one of you fix goes wrong and will cause issues elsewhere?

These are small fixes, so in this case it's not possible to break something somewhere else 📟

File: [admin]/admin2/client/main/admin_server.lua [hunk 3]
```diff
@@ -411,8 +465,27 @@ function aServerTab.onRefresh()
     guiCheckBoxSetSelected(aServerTab.AirCars, isWorldSpecialPropertyEnabled("aircars"))
     guiCheckBoxSetSelected(aServerTab.ExtraBunny, isWorldSpecialPropertyEnabled("extrabunny"))
     guiCheckBoxSetSelected(aServerTab.ExtraJump, isWorldSpecialPropertyEnabled("extrajump"))
+    guiCheckBoxSetSelected(aServerTab.RandomFoliage, isWorldSpecialPropertyEnabled("randomfoliage"))
+    guiCheckBoxSetSelected(aServerTab.SniperMoon, isWorldSpecialPropertyEnabled("snipermoon"))
+    guiCheckBoxSetSelected(aServerTab.ExtraAirResistance, isWorldSpecialPropertyEnabled("extraairresistance"))
+    guiCheckBoxSetSelected(aServerTab.UnderWorldWarp, isWorldSpecialPropertyEnabled("underworldwarp"))
+
+    triggerServerEvent("aServerRefresh", localPlayer)
```
Comment (qaisjp) [inline [admin]/admin2/client/main/admin_server.lua]:
```suggestion
    triggerServerEvent("aServerGlitchRefresh", localPlayer)
```

File: [admin]/admin2/server/admin_server.lua [hunk 4]
```diff
@@ -263,6 +263,15 @@ addEventHandler(
     end
 )
 
+addEvent("aServerRefresh", true)
```
Comment (qaisjp) [inline [admin]/admin2/server/admin_server.lua]:
```suggestion
addEvent("aServerGlitchRefresh", true)
```

File: [admin]/admin2/server/admin_server.lua [hunk 5]
```diff
@@ -263,6 +263,15 @@ addEventHandler(
...
+addEventHandler(
+    "aServerRefresh",
```
Comment (qaisjp) [inline [admin]/admin2/server/admin_server.lua]:
```suggestion
    "aServerGlitchRefresh",
```

**Comment by qaisjp** [comment]
Thank you very much!

**Pull request merged**